lms/envs/aws.py

```diff
@@ -274,6 +274,16 @@
     COMPREHENSIVE_THEME_DIR = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR')
 
 COMPREHENSIVE_THEME_DIRS = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIRS', COMPREHENSIVE_THEME_DIRS) or []
+
+# THEME_LOCALE_PATHS contain the paths to themes locale directories e.g.
+# "THEME_LOCALE_PATHS" : [
+#        "/edx/src/edx-themes/conf/locale"
+#    ],
+THEME_LOCALE_PATHS = ENV_TOKENS.get('THEME_LOCALE_PATHS', [])
+
+for theme_locale_path in THEME_LOCALE_PATHS:
+    LOCALE_PATHS += (path(theme_locale_path), )
```

Comment:
I think this can all just be:
```
LOCALE_PATHS += tuple(ENV_TOKENS.get('COMPREHENSIVE_THEME_LOCALE_PATHS', [])
```
 We will also need to work with DevOps to get a configuration_secure PR.